src/sentry/static/sentry/app/routes.jsx

```diff
@@ -1183,58 +1183,6 @@ function routes() {
             }
             component={errorHandler(LazyLoad)}
           />
-          <Route
-            path="releases/"
```

Comment:
Can we redirect these old URLs?

Reply:
We could. I'd personally vote to simplify our codebase and drop support though. Apart from a handful of opt outs, I think people shouldn't have been using these routes for months now, and I don't think we should commit to supporting these forever.

Reply:
Users could have bookmarks.

Reply:
For any route except list where it's nested under a specific version it seems unlikely due to our retention policy